sdks/js/src/index.ts

```diff
@@ -117,6 +118,10 @@ export class DustAPI {
     return this._credentials.workspaceId;
   }
 
+  setWorkspaceId(workspaceId: string) {
```

Comment:
useful in the raycast extension (as the user can change to one of their workspace on demand).

Reply:
This seems to overfit a bit on the "user oauth token" as the API key is scoped to a workspace 🤔.

Reply:
In both case, we need to specify the workspaceId because then (almost, expect /me) are made using the workspace id in the url.
Right now the problem is that the constructor overfit the workspace Api key (where you know in advance both).

It's a bit of an hack for now. Will come back to it.